src/components/file-dropzone/file-dropzone.tsx

```diff
@@ -0,0 +1,92 @@
+import { Component, h, Event, EventEmitter, Host, Prop } from '@stencil/core';
+import { FileInfo } from '../../global/shared-types/file.types';
+import { createFileInfo } from '../../util/files';
+
+/**
+ * This component enables you to seamlessly convert any region of the user interface into
+ * a file dropzone area, just by wrapping it inside the `limel-file-dropzone`.
+ *
+ * The file dropzone can then be used to allow end-users to upload files
+ * by dragging and dropping them into the specified area.
+ *
+ * After receiving the files, the component emits a `newFiles` event.
+ * The event detail would be an array of `FileInfo` objects,
+ * each representing a file dropped into the dropzone.
+ * @exampleComponent limel-example-file-dropzone
+ * @exampleComponent limel-example-file-dropzone-type-filtering
+ */
+@Component({
+    tag: 'limel-file-dropzone',
+    shadow: true,
+    styleUrl: 'file-dropzone.scss',
```

Comment:
I'm guessing this property and the empty file it points to are remnants of earlier iterations? 🙂

Reply:
Nah it's going to be filled in eventually (hopefully tomorrow) :)